src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -224,3 +224,26 @@ def app_teardown(
     )
     processor.process(force)
     return MessageResult(f"Teardown is now complete.")
+
+
+@app.command("deploy", requires_connection=True)
+@with_project_definition("native_app")
+def app_deploy(
+    **options,
+) -> CommandResult:
+    """
+    Syncs the local changes to the stage without creating or updating the application.
+    """
+
+    processor = NativeAppRunProcessor(
+        project_definition=cli_context.project_definition,
+        project_root=cli_context.project_root,
+    )
+
+    processor.build_bundle()
+    processor.process(
```

Comment:
which part deploys to the stage? is it this one?

Reply:
Yep, I'm piggybacking on [RunProcessor.process()](https://github.com/snowflakedb/snowflake-cli/blob/4198548f9b89cc5758763b7e215e8d377bbf9c13/src/snowflake/cli/plugins/nativeapp/run_processor.py#L352-L401) - see its step 3

Reply:
Doesn't NativeAppRunProcessor also create an application instance? I thought we should stop at copying files to the stage. I suggest you confirm with Cam on this.

Reply:
@sfc-gh-melnacouzi  The app creation is skipped because of the new flag `skip_app_update=True` in the run processor's `process` signature. It is also passed in as True in line 246 below.